lib/KBox/src/tasks/ADCTask.cpp

```diff
@@ -33,11 +33,18 @@ void ADCTask::loop() {
   int bat1_adc = adc.analogRead(bat1_analog, ADC_0);
   int bat2_adc = adc.analogRead(bat2_analog, ADC_0);
   int bat3_adc = adc.analogRead(bat3_analog, ADC_0);
+    
+  //permits rudder sensor on bat3 input.  Eventually should move rudder sensor calcs to a separate task
```

Comment:
I think an even better solution would be to keep ADCTask and have a config option in it to replace BAT3 by a Rudder sensor.

If the option is not set, ADCTask would send four VoltageMeasurement like it used to.
If the option is set, ADCTask would send three VoltageMeasurement and one RudderAngle.